sonic_data_client/transl_subscriber.go

```diff
@@ -186,8 +193,16 @@ func (ts *translSubscriber) notify(v *translib.SubscribeResponse) error {
 	}
 
 	spbv := &spb.Value{Notification: msg}
-	ts.client.q.Put(Value{spbv})
+	ts.client.q.EnqueueItem(Value{spbv})
 	log.V(6).Infof("Added spbv %#v", spbv)
+	err = ts.client.q.EnqueueItem(Value{spbv})
+	if st, ok := status.FromError(err); ok {
+		if st.Code() == codes.ResourceExhausted {
+			return err
+		}
+	} else {
+		log.V(6).Infof("Added spbv #%v", spbv)
```

Comment:
This log is never reached -- status.FromError() returns ok=true for nil errors.
See https://pkg.go.dev/google.golang.org/grpc/status#FromError